pctiler/pctiler/reader.py

```diff
@@ -43,8 +41,8 @@ def _get_asset_url(self, asset: str) -> str:
 
 
 @attr.s
-class MosaicSTACReader(MultiBaseReader):
-    """Simplified STAC Reader.
+class CustomSTACReader(pgstac_mosaic.CustomSTACReader):
```

Comment:
we can extent `pgstac_mosaic.CustomSTACReader` instead of creating a new custom one